automation/run.ts

```diff
@@ -89,6 +61,15 @@ export async function run(args?: string[]) {
 	// the current working dir becomes the MSYS2 homedir, so we change back.
 	process.chdir(ROOT);
 
+	if (process.platform === 'win32' && !process.env.BUILD_TMP) {
+		const randID = require('crypto')
+			.randomBytes(6)
+			.toString('base64')
+			.replace(/\+/g, '-')
+			.replace(/\//g, '_'); // base64url (RFC 4648)
+		process.env.BUILD_TMP = `C:\\tmp\\${randID}`;
```

Comment:
We did this to get around the 260-character limit of paths under Windows.

Reply:
Lets add this as a comment in the code as well, so we don't risk breaking it in the future once we forget :)